Tests/Engine/DataFeeds/LiveTradingDataFeedTests.cs

```diff
@@ -2023,7 +2023,7 @@ public void HandlesFutureAndOptionChainUniverse(SecurityType securityType)
             else if (securityType == SecurityType.Future)
             {
                 var future = algorithm.AddFuture(Futures.Indices.SP500EMini, Resolution.Minute);
-                future.SetFilter(x => x);
+                future.SetFilter(x => x.IncludeWeeklys());
```

Comment:
Can you please show the impact on the total count of contracts from this change?

Reply:
Total count of contracts is the same! 2 in 2 out!

The reason this was adjusted is the contract dates for ES we created are as follows:
12/19/19
3/19/20

But the standard contracts for those months are:
12/20/19
3/20/20

See [here](https://www.barchart.com/futures/quotes/ESZ20/historical-prices) for verification. 

Without the IncludeWeeklys() function on the filter those two dates get filtered out!